src/Components/Server/src/Builder/RazorComponentsEndpointBuilder.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Builder
+{
+    /// <summary>
+    /// <see cref="IEndpointConventionBuilder"/> to add specific component extensions.
+    /// </summary>
+    public class RazorComponentsEndpointBuilder : IEndpointConventionBuilder
+    {
+        private readonly IEndpointConventionBuilder _builder;
+
+        internal RazorComponentsEndpointBuilder(IEndpointConventionBuilder builder)
+        {
+            _builder = builder;
+        }
+
+        /// <inheritdoc />
+        public void Add(Action<EndpointBuilder> convention)
+        {
+            _builder.Add(convention);
```

Comment:
The constructor doesn't check for the _builder to not be null. So this can result in NRE.

Reply:
It's internal, we only check for null in public places where the user can pass things in.